felix/bpf-gpl/bpf.h

```diff
@@ -245,7 +245,7 @@ static CALI_BPF_INLINE void ip_dec_ttl(struct iphdr *ip)
 
 #define ip_ttl_exceeded(ip) (CALI_F_TO_HOST && !CALI_F_TUNNEL && (ip)->ttl <= 1)
 
-#if !defined(__BPFTOOL_LOADER__) && !defined (__IPTOOL_LOADER__)
+#if !defined(__BPFTOOL_LOADER__) && !defined (__IPTOOL_LOADER__) && !defined(CALI_F_XDP)
```

Comment:
Removed `__IPTOOL_LOAD__`.
However, er need the check for `!defined(CALI_F_XDP)` since we don't want to include the code related to `globals` in the XDP code.